### DISCUSSION TITLE
Want to build a one way notification chatbot in teams

### DISCUSSION THREAD

**Discussion by harshdamaniahd** [pnp/sp-dev-fx-webparts#2047] [Q&A] [answered]
Hi,
I would like to know, how can i build a one way notification chatbot in teams.

For eg as an Content Creator i would like to send an adaptive card to all user , which might have an image description etc.It can be short news or any office 365 learning tip.

How can i get started.

**Comment by hugoabernier** [comment, marked as answer]
@harshdamaniahd I would suggest that you redirect your question to the [Teams Dev Samples](https://github.com/pnp/teams-dev-samples), where they focus on Teams samples.

**Comment by harshdamaniahd** [comment]
ok... will post it there :)
On Wed, 22 Sep 2021 at 2:14 PM, Hugo Bernier ***@***.***>
wrote:

> @harshdamaniahd <https://github.com/harshdamaniahd> I would suggest that
> you redirect your question to the Teams Dev Samples
> <https://github.com/pnp/teams-dev-samples>, where they focus on Teams
> samples.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pnp/sp-dev-fx-webparts/discussions/2047#discussioncomment-1368902>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACZ4C6OSZNVJH6D7CIFY2KLUDHCDFANCNFSM5EQP3ZPQ>
> .
>
-- 

*With Regards,*
*Harsh Damania*


[image: https://www.facebook.com/damaniaharsh]
<https://www.facebook.com/damaniaharsh>   [image: Twitter]
<https://twitter.com/harshdamaniahd>   [image: Sharepoint]
<http://sharepointknowledgecenter.blogspot.in/>  [image: Linkedin]
<https://www.linkedin.com/profile/view?id=119418247>
